### PR TITLE
Added the set command and reference to mysql docs

### PR DESCRIPTION
**Pull request by aleon1220**
added link to mysql docs
added set command to make it explicit where it has to run

### CODE REVIEW

File: setup/productionize/persistence/orca-sql.md [hunk 1]
```diff
@@ -27,7 +27,13 @@ The `migration` user will only be used to perform schema changes on the database
 
 Before deploying Orca, the schema and database uses must first be manually setup:
 
-1. Set MySQL Server `tx_isolation` setting to `READ-COMMITTED`
+1. Set MySQL Server variable `tx_isolation` setting to `READ-COMMITTED`. Refer to [MySQL Server System Variables](https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_tx_isolation)
```
Comment (dorbin) [inline setup/productionize/persistence/orca-sql.md]: This page times out for me. Also, there's a version number in the url. Will the link become obsolete?

Comment (dorbin) [inline setup/productionize/persistence/orca-sql.md]:
```suggestion
1. Set MySQL Server variable `tx_isolation` setting to `READ-COMMITTED`. Refer to [MySQL Server System Variables](https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_tx_isolation).
```

Comment (aleon1220) [reply setup/productionize/persistence/orca-sql.md]: yes is for version 5.7 only

Comment (dorbin) [reply setup/productionize/persistence/orca-sql.md]: Thanks, I hadn't noticed the heading above, _supposed to be_ for 5.7 only. :)

And the url now works for me.

**Review by dorbin — APPROVED** [review]

**Pull request merged**